### DISCUSSION TITLE
Is there an 1.2.3 ipa file iOS release for the master branch that can download for iPhone outside the apple store?

### DISCUSSION THREAD

**Discussion by plia7** [rustdesk/rustdesk#7186] [Q&A] [answered]
Hello,

It looks like the current master branch (version 1.2.3) has newer code than the one that’s available on apple store (versions 1.2.2).
Is there an actual ipa file or another file type that I could download from RustDesk’s github page and sideload it on my iPhone without the need to wait for the newer version to get published on apple store or building the master branch myself? I’m able to sideload iPAs and deb files on my phone.

Thank you.

**Comment by tschettervictor** [comment, marked as answer]
I don’t think so. We will have to wait for the TestFlight builds. I hope they can be pushed soon.
